### ISSUE TITLE
Combine two languages

### ISSUE THREAD

**Issue by bikmetle** [RajSolai/TextSnatcher#52]
Is there a way to combine two languages? Due to the fact that the russian language is poor, we often write words in English. And during OCR, the program cannot correctly recognize English, because the language for the main text is russian

**Comment by jheld** [comment]
Tesseract has multilingual support on a given image, so, in principle I'd assume it's possible to leverage that via this tool with some change.

**Comment by RajSolai** [comment]
Yes this can be done, the project is dormant at this stage as I'm saving for a PC for my Linux development I will take this up sure when I restart
